src/SortPeopleByDateOfBirth.java

```diff
@@ -0,0 +1,25 @@
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortPeopleByDateOfBirth {
+
+  static void sortPeopleByDateOfBirth(){
+    List <Person> people = List.of(
+        new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@gmail.com"),
+        new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@gmail.com"),
+        new Person("Harry", LocalDate.parse("1980-09-25"), "+447220012555", "harry@gmail.com"),
+        new Person("Anne", LocalDate.parse("1969-03-15"), "+447220012222", "anne@gmail.com"),
+        new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@gmail.com"));
+
+    people.sort(Comparator.comparing(Person::getDateOfBirth));
+
+    people.forEach(
```

Comment:
Instead of printing the output of this list here, perhaps its better to return the sorted list instead and then performing the loop?